src/text-input/__tests__/TextInput.test.js

```diff
@@ -51,60 +47,56 @@ describe('TextInput', () => {
       )
     }
 
-    const { getByDisplayValue, getByTestId } = render(<ControlledTextInput />)
-    const input = getByTestId('input')
+    render(<ControlledTextInput />)
+    const input = screen.getByTestId('input')
     userEvent.click(input)
 
     expect(document.activeElement).toEqual(input)
     userEvent.type(input, 'Testing')
-    expect(getByDisplayValue('Testing')).toEqual(input)
+    expect(screen.getByDisplayValue('Testing')).toEqual(input)
   })
 
   it('should not be interactive if `disabled` is passed in', () => {
-    const { getByDisplayValue, getByTestId } = render(makeTextInputFixture({ disabled: true }))
-    const input = getByTestId('input')
+    render(makeTextInputFixture({ disabled: true }))
+    const input = screen.getByTestId('input')
     userEvent.type(input, 'Testing')
 
-    expect(() => getByDisplayValue('Testing')).toThrowError()
-    expect(getByDisplayValue('')).toEqual(input)
-  })
-})
-
-describe('TextInputField', () => {
```

Comment:
Most of these tests were already in `TextInputField.test.js`, but must not have been removed when the files were split up.